Packs/RDPCacheHunting/Playbooks/playbook-RDP_Bitmap_Cache_-_Detect_and_Hunt.yml

```diff
@@ -0,0 +1,1525 @@
+id: RDP Bitmap Cache - Detect and Hunt
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: RDP Bitmap Cache - Detect and Hunt
+description: This playbook automates the collection and forensic analysis of RDP sessions cache data by collecting the cache files and convert it to an image, extract readable text from the image and build IOC's from text and finally enrich any extracted indicators for further hunting
```

Comment:
Use MD instead.
I see you also used the Retrieve File from Endpoint - Generic V3 playbook.
That means that we need to document in the playbook's readme that we can collect using x,y,z.